### PR TITLE
XSS fix, cont.

### PR DESCRIPTION
**Pull request by PipoCanaja**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Review by laf — APPROVED** [review]
LGTM

File: includes/html/pages/addhost.inc.php [hunk 1]
```diff
@@ -45,9 +45,9 @@
...
                 'snmp_disable' => 1,
-                'os'           => $_POST['os'] ? $_POST['os_id'] : 'ping',
-                'hardware'     => $_POST['hardware'],
-                'sysName'      => $_POST['sysName'],
+                'os'           => $_POST['os'] ? strip_tags($_POST['os_id']) : 'ping',
+                'hardware'     => strip_tags($_POST['hardware']),
+                'sysName'      => strip_tags($_POST['sysName']),
             ];
         } elseif ($_POST['snmpver'] === 'v2c' || $_POST['snmpver'] === 'v1') {
             if ($_POST['community']) {
```
Comment (PipoCanaja) [inline includes/html/pages/addhost.inc.php:53]: Is Config::set safe in term of SQL injection etc etc?

Comment (laf) [reply includes/html/pages/addhost.inc.php:53]: I don't think Config::set actually stores anything long term does it? Especially in addhost

Comment (PipoCanaja) [reply includes/html/pages/addhost.inc.php:53]: You are right. It does set only to allow addhost to try the communities and find the right one.

**Pull request merged**